Week3/homework/js-exercises/fortuneTeller.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+function tellFortune(number,partner,location,job){
+return  `In ${time[Math.floor(Math.random()*5)]}, You will be a ${job[Math.floor(Math.random()*5)]} in ${location[Math.floor(Math.random()*5)]}, and married to ${partner[Math.floor(Math.random()*5)]} with ${number[Math.floor(Math.random()*5)]} kids.`
+}
+let time =[2020,2022,2024,2026,2028];
+let myChildren = [3,4,5,6,7];
+let myWife = ['Fatime','Patigul','Patime','Aypatem','Patemhan'];
```

Comment:
And don't forget to use `s` at the end of the variable name, to show that there are multiple elements in the array